modules/cluster-logging-updating-logging-to-current.adoc

```diff
@@ -0,0 +1,219 @@
+// Module file include in the following assemblies:
```

Comment:
Need to add a metadata line with the name of the assembly or assemblies in which this file is included. E.g.:

`// filename.adoc`